protos/core.proto

```diff
@@ -22,6 +22,7 @@ message RunConfig {
   bool include_dependencies = 3;
   bool full_refresh = 2;
   bool carry_on = 4;
+  repeated string tags = 5;
```

Comment:
nit: suggest putting this just below `actions` since it is quite related!

Reply:
Done